src/parse-templates.ts

```diff
@@ -71,7 +83,7 @@ function isEscaped(template: string, _offset: number | undefined) {
  *
  * @param template The template to parse
  * @param relativePath Relative file path for the template (for errors)
- * @param templateTag Optional template tag if parsing template tags is enabled
+ * @param options Optional {ParseTemplateOptions}
```

Comment:
Since this is in TS, it doesn't need the JSDoc style attribute, but it's worth documenting anyway:

```suggestion
 * @param options optional configuration options for how to parse templates
```